CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.86 (December 10, 2021)
+
+### Dependency Upgrades
+
+- `TwilioVideo` has been updated from 4.6.2 to 4.6.3. [#180](https://github.com/twilio/twilio-video-app-ios/pull/180)
+
+-----------
+
```

Comment:
```suggestion

## 0.86 (December 10, 2021)

- Updated repo to use Fastlane match for provisioning.

-----------
```

Reply:
We already used up 0.86 and forgot to update changelog + release. It was basically an internal change that doesn't really impact customers.